diktat-rules/src/test/kotlin/org/cqfn/diktat/util/TestUtils.kt

```diff
@@ -97,13 +95,7 @@ internal fun format(ruleSetProviderRef: (rulesConfigList: List<RulesConfig>?) ->
             fileName = fileName.removeSuffix("_copy"),
             script = fileName.removeSuffix("_copy").endsWith("kts"),
             cb = cb,
-            userData = mapOf("file_path" to fileName.removeSuffix("_copy")),
-        ),
-        VisitorProvider(
-            ruleSets = ruleSets,
             debug = true,
-            // setting this to `true` breaks smoke test
-            isUnitTestContext = false,
```

Comment:
need to check how it did used before

Reply:
it was used in `VisitorProvider` with comment:
```kotlin
// When running unit tests,the RunAfterRule annotation is ignored. The code provided in the unit should
// be formatted as if the rule on which it depends was already applied. In this way the unit test can be
// restricted to one single rule instead of having to take into account all other rules on which it
// might depend.
```
Now it's not available, this property moved to internal class and `false` by default